packaging/torchtext/meta.yaml

```diff
@@ -15,12 +15,14 @@ requirements:
     - python
     - setuptools
     - cpuonly
+    - pytorch-mutex 1.0 cpu  # [not osx ]
```

Comment:
Can you elaborate a bit more as to why we are adding this to the host?

Reply:
It may not be required for host. I was mostly following the template in torchvision and [this PR](https://github.com/pytorch/builder/pull/823). The addition to host was from torchvision and I'm throwing things at the wall -- certainly not as principled as I'd like...